src/group/urls.py

```diff
@@ -5,5 +5,8 @@
 urlpatterns = [
     path('', views.index, name='index_group'),
     path('add_group', views.add_group, name='add_group'),
-    path('groups_list', views.groups_list, name='groups_list')
+    path('groups_list', views.groups_list, name='groups_list'),
+    path('edit_group/<int:id>', views.edit_group, name='edit_group'),
+    path('delete_group/<int:id>', views.delete_group, name='delete_group'),
```

Comment:
Since you already include application urls in the project url, consider specifying only actions here to get uniform and simplified urls:
```
    path('list/', views.groups_list, name='groups_list'),
    path('edit/<int:id>', views.edit_group, name='edit_group'),
    path('delete/<int:id>', views.delete_group, name='delete_group'),
```

Reply:
Ok, thanks.

Reply:
Fixed!